Formula/voronota.rb

```diff
@@ -0,0 +1,48 @@
+class Voronota < Formula
+  desc "Compute Voronoi diagram vertices for macromolecular structures"
+  homepage "https://github.com/kliment-olechnovic/voronota"
+  url "https://github.com/kliment-olechnovic/voronota/archive/refs/tags/v1.29.4370.tar.gz"
+  sha256 "c7b55f1de0a36502121d87d5fe980dbc13223e0323c86a644cf1f3b39de52eda"
+  license "MIT"
+  version "1.29.4370"
+  head "https://github.com/kliment-olechnovic/voronota.git", branch: "master"
+
+#   option "with-js",      "Enable Voronota-JS expansion"
+#   option "with-lt",      "Enable Voronota-LT expansion"
+#   option "with-gl",      "Enable Voronota-GL expansion"
+#   option "with-openmp",  "Compile with OpenMP support"
+#   option "with-mpi",     "Compile with MPI support"
```

Comment:
Is it possible to include all the options by default? If possible, it would be desirable.